src/reducers/app.js

```diff
@@ -63,10 +72,37 @@ function hasZoomedViaMousewheel(state: boolean = false, action: Action) {
       return state;
   }
 }
+
+function isSidebarOpenPerPanel(
+  state: IsSidebarOpenPerPanelState,
+  action: Action
+): IsSidebarOpenPerPanelState {
+  if (state === undefined) {
+    state = {};
+    tabSlugs.forEach(tabSlug => (state[tabSlug] = false));
+  }
```

Comment:
Because the computation of the default state isn't simple, I decided to do it here instead of using the default arguments like elsewhere.

Reply:
I'd say that is totally okay until we find a cleaner solution :)